tgui/.gitignore

```diff
@@ -1,7 +1,21 @@
+## NPM garbage
 node_modules
 *.log
 package-lock.json
 
-/packages/tgui/public/.tmp/**/*
-/packages/tgui/public/**/*.hot-update.*
-/packages/tgui/public/**/*.map
+## Yarn stuff
+/.pnp.*
+/.yarn/*
+!/.yarn/releases
+!/.yarn/plugins
+!/.yarn/sdks
+!/.yarn/versions
+!/.yarn/lock.yml
+
+## Build artifacts
+/public/.tmp/**/*
+/coverage
+
+## Previously ignored locations that are kept to avoid confusing git
+## while transitioning to a new project structure.
+/packages/tgui/public/**
```

Comment:
there will be a followup PR to change this later?

Reply:
No, this will remain as a legacy function to stop kneecapping older branches